### PR TITLE
Drawer screen re-design (Material design)

### LINKED ISSUE #628: Bug: Android: Drawer UI overlaps when parachins section is expanded.

### PR DESCRIPTION
**Pull request by silva-fj**
This PR removes the usage of UI Kitten components as well as follows material design guidelines for drawers (Note that there are no expandable items).

As shown in the screenshots with the components from react-native-paper we can see what screen is currently active.

Screenshot 2021-11-28 at 1 30 37 PM

Screenshot 2021-11-28 at 1 31 19 PM

Also Fixes #628

### CODE REVIEW

**Comment by silva-fj** [comment]
FYI @stephanie-olp

**Review by ishrak-imam — DISMISSED** [review]

**Comment by geastwood** [comment]
How many screens we have left to adapt?

File: src/screen/Drawer/DrawerScreen.tsx [hunk 1]
```diff
@@ -20,126 +19,156 @@ import {
...
+      </View>
+      <Divider />
+      <ScrollView>
+        <Drawer.Section>
+          <Drawer.Item
+            label="Dashboard"
+            icon="view-dashboard"
+            active={activeScreen === 'Dashboard'}
+            onPress={() => {
+              setActiveScreen('Dashboard');
```
Comment (ahoseinian) [inline src/screen/Drawer/DrawerScreen.tsx]:
```suggestion
            active={activeScreen === dashboardScreen}
            onPress={() => {
              setActiveScreen(dashboardScreen);
```
also for other routes

**Review by ahoseinian — DISMISSED** [review]

**Comment by silva-fj** [comment]
> How many screens we have left to adapt?

We are making small steps to migrate gradually, most of the screens are already using the new fonts, colors and so on, then we are going to start replacing `Card`s and other components. So it's not per screen but per component. This one was an exception.

**Review by ahoseinian — APPROVED** [review]

**Pull request merged**